Tweeter.Tests/DAL/TweeterRepoTests.cs

```diff
@@ -110,5 +128,58 @@ public void RepoEnsureUsernameExistsOfTwit()
             // Assert
             Assert.IsNotNull(found_twit);
         }
+
+        [TestMethod]
+        public void EnsureCanGetAllTweets()
+        {
+            Tweet tweet1 = new Tweet { TweetId = 1, Message = "Hi" };
+            Tweet tweet2 = new Tweet { TweetId = 2, Message = "Yo" };
+            Tweet tweet3 = new Tweet { TweetId = 3, Message = "Boo"};
+            Repo.Add(tweet1);
+            Repo.Add(tweet2);
+            Repo.Add(tweet3);
+
+            int expected_count = 3;
+            List<Tweet> all_tweets = Repo.GetAllTweets();
+            int actual_count = all_tweets.Count;
+
+            Assert.AreEqual(expected_count, actual_count);
+        }
+
+        [TestMethod]
+        public void EnsureCanAddNewTweet()
+        {
+            
+
+            Tweet tweet1 = new Tweet {TweetId = 1, Message = "Hi" };
+            Tweet tweet2 = new Tweet { TweetId = 2, Message = "Yo" };
+            Repo.Add(tweet1);
+            Repo.Add(tweet2);
+
+            int expected_count = 2;
+            List<Tweet> all_tweets = Repo.GetAllTweets();
+            int actual_count = all_tweets.Count;
+
+            Assert.AreEqual(expected_count, actual_count);
+        }
+
+        [TestMethod]
+        public void EnsureCanDeleteATweet()
+        {
+            
+
+            Tweet tweet1 = new Tweet { TweetId = 1, Message = "Hi"};
+            Tweet tweet2 = new Tweet { TweetId = 2, Message = "Yo" };
+            Repo.Add(tweet1);
+            Repo.Add(tweet2);
+
```

Comment:
While this test works, suppose the previous test (where `expected_count` is 2) didn't exist... it's nice to do a "sanity check" within your test method by asserting that the count after adding `tweet1` and `tweet2` is 2, and then asserting the count is 1 after removing `tweet1`.